### ISSUE TITLE
[BUG][FormTribe 🔥][50 Points] The Set Form Brick Installation Guide Responsiveness issue

### ISSUE THREAD

**Issue by itzabhinavarya** [formbricks/formbricks#1014]
### Issue Summary

Installation guide of "Set form brick is not responsive"
<img width="559" alt="image" src="https://github.com/formbricks/formbricks/assets/95561280/4d98383f-6c8e-4334-b886-b66f8430c8e1">

### Steps to Reproduce

1. Head over the Set FormBricks up in minutes
2. The Code setion is not responsive 

### Expected behavior

It should looks like this

<img width="440" alt="image" src="https://github.com/formbricks/formbricks/assets/95561280/75b7a020-ff9f-446a-a3df-2122286ac926">
 

### Environment

- [ ] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS: Not required
- Node: Not required
- npm: Not required
```

**Comment by itzabhinavarya** [comment]
@jobenjada  Sir, Please assign this issue to me.

**Comment by jobenjada** [comment]
@itzabhinavarya its yours, pls go for it !

**Comment by itzabhinavarya** [comment]
@jobenjada Sir, I already have solved this issue. and the PR regarding this issue already got merged by @ShubhamPalriwala Sir. 
PR no. #1018 
Please check this for reference.

**Comment by ShubhamPalriwala** [comment]
Closed via #1018
